### PR TITLE
ZEN-23557 ZenOperator role to change Production state Changes

### PR DESCRIPTION
**Pull request by ssOlexBaiko**

### CODE REVIEW

File: Products/Zuul/routers/device.py [hunk 1]
```diff
@@ -317,7 +317,9 @@ def setInfo(self, **data):
         @rtype: DirectResponse
         """
         facade = self._getFacade()
-        if not Zuul.checkPermission('Manage Device', self.context):
+        if not (Zuul.checkPermission('Manage Device', self.context) or (
```
Comment (glujan) [inline Products/Zuul/routers/device.py]: This permission is defined in `Products/ZenModel/ZenossSecurity.py` - why use magic string instead? Same for `"Change Device Production State"`.

File: Products/ZenUI3/browser/resources/js/zenoss/DeviceOverviewPanel.js [hunk 2]
```diff
@@ -624,7 +624,7 @@
             text: _t('Save'),
             xtype:'button',
             ref: '../savebtn',
-            disabled: true,
+            disabled: Zenoss.Security.doesNotHavePermission('Change Device Production State'),
```
Comment (glujan) [inline Products/ZenUI3/browser/resources/js/zenoss/DeviceOverviewPanel.js]: What happens if user has `ZEN_MANAGE_DEVICE` perm but does not have this one? Is it intentional?

**Comment by ssOlexBaiko** [comment]
retest this

**Comment by ssOlexBaiko** [comment]
retest this

**Comment by glujan** [comment]
Retest this

**Comment by ssOlexBaiko** [comment]
please retest this

**Pull request merged**